### ISSUE TITLE
Picker modal is missing basic accessibility features

### ISSUE THREAD

**Issue by dweldon** [filestack/filestack-js#295]
There are likely numerous improvements to be made, but the most glaring that I see are:

1. The modal doesn't trap focus
2. The "buttons" are spans and can't be tabbed to
3. The "buttons" don't have outlines
4. The containing div doesn't have a role of `dialog`
5. The containing div should have an `aria-labelledby` and an optional `aria-describedby`

**Comment by sarimarton** [comment]
Why is it closed?? This issue came up in our product as a bug, clients demand it to be fixed.

**Comment by dweldon** [comment]
Because, I believe, someone submitted a PR for something vaguely related to a11y and it was accepted. The fact remains that the picker isn't even remotely accessible. Just turn on ChromeVox and try to use it. Can't? Oh that's right... because the dialog doesn't gain or trap focus.

Lack of basic accessibility in the picker UI should be a showstopping bug, but it doesn't appear to be a priority for dev or management (I've written to both about this).

**I say this as someone who has paid a lot of money for Filestack over many years. This nonsense is incredibly frustrating.** If I were shopping around for another solution, I'd recommend looking elsewhere.

**Comment by sarimarton** [comment]
Workaround:

```
   .pick(
        {
            onOpen: () => {
                const observer = new MutationObserver(() => {
                    const el = document.querySelector('#__filestack-picker [tabindex]');
                    if (el) {
                        el.focus();
                        observer.disconnect();
                    }
                });
                observer.observe(document.body, { childList: true, subtree: true });
            },
        // ...
```
